server/middleware/setUpWebSecurity.ts

```diff
@@ -40,7 +40,12 @@ export default function setUpWebSecurity(): Router {
           formAction: ["'self'", new URL(config.apis.hmppsAuth.url).hostname],
           // Hash allows inline script pulled in from https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/template.njk
           scriptSrc,
-          styleSrc,
+          styleSrc: [
+            "'self'",
+            'code.jquery.com',
+            "'unsafe-inline'",
+            (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`,
+          ],
```

Comment:
this should probably remain as-is as the contents of `styleSrc` are already the first 3 entries in this array

add the function as the last entry on line 23

https://github.com/ministryofjustice/hmpps-jobs-board-ui/blob/main/server/middleware/setUpWebSecurity.ts#L23